Docs/term entry codepointat (#7071)
Codecademy/docs: content/javascript/concepts/strings/terms/codePointAt/codePointAt.md

---
Title: '.codePointAt()'
Description: 'Returns the Unicode code point value at a specified index position in a string, including complex characters like emojis and symbols.'
Subjects: 
  - 'Computer Science'
  - 'Web Development'
Tags:
  - 'Unicode'
  - 'Strings'
  - 'Methods'

>   - 'Text Processing'

Reviewer: ```suggestion
  - 'Methods'
  - 'Strings'
  - 'Text Processing'
  - 'Unicode'
```

CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-developer'
---

> The **`.codePointAt()`** [method](https://www.codecademy.com/resources/docs/javascript/methods) is a JavaScript string method that returns a non-negative integer that is the Unicode code point value of the character at a specified index position. Unlike the older method [`.charCodeAt()`](https://www.codecademy.com/resources/docs/javascript/concepts/strings/charCodeAt/charCodeAt.md), it properly handles the full spectrum of Unicode characters, including emojis, symbols, and characters from any language.

Reviewer: ```suggestion
The **`.codePointAt()`** [method](https://www.codecademy.com/resources/docs/javascript/methods)is a JavaScript string method that returns a non-negative integer representing the Unicode code point of the character at a specified index. Unlike the older [.charCodeAt()](https://www.codecademy.com/resources/docs/javascript/concepts/strings/charCodeAt/charCodeAt.md) method, .codePointAt() accurately handles the full range of Unicode characters including emojis, symbols, and characters from any language.
```

## Syntax

> ```js

Reviewer: ```suggestion
```pseudo
```
